guidelines/content-guidelines/01-content-strategy-k2.md

```diff
@@ -0,0 +1,135 @@
+
+### Troubleshooting topics
+
+Provide a condition that the reader may want to correct, followed by one or more descriptions of its cause and suggested remedies.
+
+It's good practice to use three standard headlines (like “Condition”, “Cause”, “Remedy”), each might have just one sentence or more as needed. For remedy, use a numbered list if there are multiple steps to follow, and a bullet list if there are several equally valid solutions.
+
+> In its simplest form, troubleshooting information follows this pattern:
+>
+> 1. A condition or symptom. Usually the condition or symptom is an undesirable state in a system, a product, or a service that a reader wants to correct.
+> 2. A cause for the condition or symptom.
+> 3. A remedy for the condition or symptom.
+>
+> The troubleshooting topic provides sections for describing the condition, causes, and remedies needed to restore a system, a product, or a service to normal.
+>
+> For some conditions there could be more than one cause-remedy pair. The troubleshooting topic accommodates this. Typically, a cause is immediately followed by its remedy. Multiple cause-remedy pairs can provide a series of successive fall-backs for resolving a condition.
+>
+> Cause and remedy might occur in combinations other than pairs. It is possible to have:
+>
+> * Multiple causes with the same remedy
+> * A single cause with more than one remedy
+> * A remedy with no known cause
+> * A cause with no known remedy
+> The troubleshooting information type also can be used to document alarm clearing strategies.
+> [- DITA Troubleshooting Topics](http://docs.oasis-open.org/dita/dita/v1.3/errata02/os/complete/part3-all-inclusive/archSpec/technicalContent/dita-troubleshooting-topic.html)
+
+## Graphics
+
+### Architecture and Flow Diagrams
+
+*Klaudia is preparing the new diagram style, see <https://github.com/kyma-project/community/issues/542>.*
+
+### Screenshots
+
+Wherever possible, present screenshots as SUI (simplified user interfaces). Basically, this means to blur out all UI elements that aren't essential for the task at hand.
+
+For more information, see:
+• <https://www.techsmith.com/blog/simplified-user-interface/>
+• <https://www.tcworld.info/e-magazine/technical-writing/simplified-graphics-and-screenshots-in-software-documentation-1102/>
+
+## Documentation Structure and Target Groups
+
+On the Kyma website, we’ll have five main tabs containing multiple documents each, plus a glossary.
+
+### Overview
+
+Target Group: Decision Makers (Tech Leads) and newbies.
+Contains a quick overview of the idea behind Kyma, a diagram of the main areas with a brief explanation, and, as needed, longer documents going into the details of each main area.
+
+### Getting Started
+
+Target Group: Software Developers who quickly want to see what they can do with Kyma.
+Contains a guide/tutorial designed by PB.
```

Comment:
😆

Reply:
I'll do my best to remember updating this before we go live!

Reply:
Updated! :D